cves/CVE-2013-2251.yml

```diff
@@ -205,4 +204,5 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: Inputs were not sanitized and defense in depth was not applied. This
+          lead to remote code execution and is completely unacceptable.
```

Comment:
More information is needed here. Why is it unacceptable? What lessons can be learned for the future? How can future vulnerabilities be avoided?

Reply:
Information added